### PR TITLE
Remove useLayout

### LINKED ISSUE #1067: Layouts break when resizing the window
image

### PR DESCRIPTION
**Pull request by loucass003**
This will remove the use of useLayout that was handling the size of most layouts with javascript. This was slow and also didnt work all the time when resizing the window. I removed that function and redid all the layouts with css only.
I also slided in minor css tweaks for padding and alignments, mostly for mobile

Fixes #1067

### CODE REVIEW

File: gui/src/components/MainLayout.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.main-layout {
+    display: grid;
```
Comment (TheDevMinerTV) [inline gui/src/components/MainLayout.scss:2]: Why is this not in tailwind?

Comment (ImUrX) [reply gui/src/components/MainLayout.scss:2]: i think it makes sense in this case, because it needs to be a grid for the grid template to be applied

**Review by ImUrX — CHANGES_REQUESTED** [review]
make prettier check the scss pls

File: gui/src/components/onboarding/pages/ConnectTracker.tsx [hunk 2]
```diff
@@ -146,9 +141,9 @@ export function ConnectTrackersPage() {
   );
 
   return (
-    <div className="flex flex-col h-full items-center px-4 pb-4">
-      <div className="flex gap-10 mobile:flex-col w-full xs:max-w-7xl">
-        <div className="flex flex-col w-full xs:max-w-sm">
+    <>
```
Comment (ImUrX) [inline gui/src/components/onboarding/pages/ConnectTracker.tsx:144]: is this even necessary?

File: gui/src/components/settings/pages/Serial.tsx [hunk 3]
```diff
@@ -100,7 +89,8 @@ export function Serial() {
         setSerialOpen(true);
       }
 
-      if (data.log && consoleRef.current) {
+      if (data.log) {
+        console.log(data.log);
```
Comment (ImUrX) [inline gui/src/components/settings/pages/Serial.tsx:93]: nani
```suggestion
```

File: gui/src/components/settings/pages/Serial.tsx [hunk 4]
```diff
@@ -124,6 +114,8 @@ export function Serial() {
       consoleRef.current.scrollTo({
         top: consoleRef.current.scrollHeight,
       });
+
+    console.log(consoleContent);
```
Comment (ImUrX) [inline gui/src/components/settings/pages/Serial.tsx:118]:
```suggestion
```

File: gui/src/components/settings/pages/Serial.tsx [hunk 5]
```diff
@@ -207,69 +199,49 @@ export function Serial() {
...
+              >
+                {l10n.getString('settings-serial-factory_reset')}
+              </Button>
+              <Button variant="quaternary" onClick={getInfos}>
+                {l10n.getString('settings-serial-get_infos')}
+              </Button>
+              <Button variant="quaternary" onClick={getWifiScan}>
+                {l10n.getString('settings-serial-get_wifi_scan')}
+              </Button>
+              {/* {isMobile && ( */}
```
Comment (ImUrX) [inline gui/src/components/settings/pages/Serial.tsx:232]:
```suggestion
```

**Review by ImUrX — CHANGES_REQUESTED** [review]

**Review by ImUrX — CHANGES_REQUESTED** [review]
tracker assignment tracker list seems to be broken. no longer shows unassigned trackers and assigned trackers in different lists

**Review by ImUrX — APPROVED** [review]

**Pull request merged**